app/scripts/controllers/transactionsBodyModal.js

```diff
@@ -1,16 +1,49 @@
-import { beautifyIndent, returnContentType } from '../utils'
+import { beautifyIndent, returnContentType, retrieveBodyProperties } from '../utils'
+import { Api } from '../services/rest'
 
-export function TransactionsBodyModalCtrl ($scope, $uibModalInstance, bodyData) {
+export function TransactionsBodyModalCtrl ($scope, $uibModalInstance, config, Alerting, bodyData) {
   $scope.bodyData = bodyData
+  const defaultLengthOfBodyToDisplay = config.defaultLengthOfBodyToDisplay
 
+  $scope.partialBody = false
```

Comment:
Following on from above, this can't be defaulted to false as at this point as the request has already been executed outside of the modal and the modal need to be passed this information.